ui/src/app/features/user-table/user-table.component.ts

```diff
@@ -42,4 +42,8 @@ export class UserTableComponent extends TableComponent implements OnInit {
   delete(user: AppUser): void {
     this.deleteUser.emit(user);
   }
+
+  isSerched():boolean{
```

Comment:
please fix name typo, change method implementation to check by the search text not ng-reflect-model class